lms/static/sass/views/_bookmarks.scss

```diff
@@ -0,0 +1,141 @@
+// Rules for placing bookmarks and search button side by side
+.wrapper-course-modes {
+  border-bottom: 1px solid lightgray;
+  display: table;
+  width: 98%;
+  padding: 6px 3px;
+
+  > div {
+    display: table-cell;
+  }
+}
+
+
+// Rules for Bookmarks Button
+.courseware-bookmarks-button {
+  width: 40%;
+  padding: 0 3px;
+  vertical-align: top;
+
+  button[aria-pressed="true"] {
+    &:before {
+      content: "\f02e";
+      font-family: FontAwesome;
+    }
+
+    background-color: $white;
+  }
+
+  button[aria-pressed="false"] {
+    &:before {
+      content: "\f097";
+      font-family: FontAwesome;
+    }
+
+    background: none;
+  }
+
+  .bookmarks-button {
+    background: none;
+    border-radius: ($baseline/4);
+    padding: 9px 8px 10px;
+    width: 100%;
+
+    &:hover {
```

Comment:
In general we avoid the use of !important. Is there a specificity issue here that makes these designations critical?